freetype/__init__.py

```diff
@@ -650,8 +650,7 @@ def get_cbox(self):
         which is dedicated to this single task.
         '''
         bbox = FT_BBox()
-        error = FT_Outline_Get_CBox(byref(self._FT_Outline), byref(bbox))
-        if error: raise FT_Exception(error)
+        FT_Outline_Get_CBox(byref(self._FT_Outline), byref(bbox))
```

Comment:
As in the commit message! FT_Outline_Get_CBox does not return a value! It segfault as you wrote it!